lib/server_health_check_rails/logger.rb

```diff
@@ -0,0 +1,63 @@
+# The health_index_path route is pinged frequently. We don't want that to clutter
+# our main log file, so we intercept all calls to that route and change the logger.
+module ServerHealthCheckRails
```

Comment:
See my comment above, as it applies here as well. These customizations are specific to us and our use of this gem, not a general audience.